add possibility to use action for multiple resources (#15)
telekom-mms/terratest-action: README.md

#### azure

test case

* *Exists

>   * description: check if terraform resource exist
>   * explanation: a true / false boolean check if the resource could be created

Reviewer: ```suggestion
  * description: check if the terraformresource was created. Returns true or false
```

* *Show

>   * description: check if created terraform resource matches the conditions that should be checked
>   * explanation: runs a terraform show of the created resource and checks if the conditions matches the conditions defined over `options`

Reviewer: ```suggestion
  * description: runs a terraform show on the created resource and checks if the results matches the test-conditions defined in `options`
```
